mqtt.kafka.asyncapi.proxy/docker/compose/docker-compose.yaml

```diff
@@ -14,6 +14,8 @@ services:
       ZILLA_INCUBATOR_ENABLED: "true"
     volumes:
       - ../../zilla.yaml:/etc/zilla/zilla.yaml
+      - ../../mqtt/asyncapi.yaml:/etc/zilla/mqtt/mqtt-asyncapi.yaml
+      - ../../kafka/asyncapi.yaml:/etc/zilla/kafka/kafka-asyncapi.yaml
```

Comment:
It is better to just use the naming schemes `*-asyncapi.yaml` since that aligns with how AsyncAPI has their examples. This way we can remove the `mqtt` and `kafka` folders from both sides of these mappings.

Renaming all references to `mqtt-asyncapi.yaml` and `kafka-asyncapi.yaml`, including the filenames in the example, will make the setup more straightforward and easier to understand for users.